geomesa-gt/geomesa-gt-partitioning/src/test/scala/org/locationtech/geomesa/gt/partition/postgis/PartitionedPostgisDataStoreTest.scala

```diff
@@ -109,14 +109,37 @@ class PartitionedPostgisDataStoreTest extends Specification with BeforeAfterAll
   }
 
   "PartitionedPostgisDataStore" should {
+    "not work" in {
+      val ds = DataStoreFinder.getDataStore(params.asJava)
+      ds must not(beNull)
+
+      try {
+        ds must beAnInstanceOf[JDBCDataStore]
+
+        // This sft name exceeds 31 characters, so it should fail
+        val sft = SimpleFeatureTypes.renameSft(this.sft, "abcdefghijklmnopqrstuvwxyzabcde_____")
+        ds.getTypeNames.toSeq must not(contain(sft.getTypeName))
+        ds.createSchema(sft) must throwAn[java.io.IOException].like {
+          case e => e.getMessage mustEqual "Error occurred creating table"
```

Comment:
could we check the e.getCause, or is there some other way we can verify that our custom error message is showing up here?